project/Dependencies.scala

```diff
@@ -25,7 +25,7 @@ object Dependencies {
   val jacksonVersion = "2.9.9"
   val jacksonDatabindVersion = "2.9.9.1"
 
-  val scala212Version = "2.12.8"
+  val scala212Version = "2.12.9-bin-d386f96"
```

Comment:
Do we do that? upgrade to a `bin-*` ?

Reply:
This is a draft PR just to see what CI thinks ;)
We'll update and mark 'Ready for review' when 2.12.9 is actually out